docs/helpers.asciidoc

```diff
@@ -0,0 +1,99 @@
+[[Helpers]]
+== Client Helpers
+
+The Elasticsearch Ruby client includes some useful helpers for a more comfortable experience with some APIs.
+
+=== Bulk Helper
+
+This helper aims to provide a nicer developer experience when using the Bulk API. At its simplest, you can send it a collection of Hashes in an Array, and it will bulk ingest them into Elasticsearch.
```

Comment:
```suggestion
This helper provides a better developer experience when using the Bulk API. At its simplest, you can send it a collection of hashes in an array, and it will bulk ingest them into {es}.
```